library/src/main/res/values/library_context_menu.android_strings.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<resources>
+	<string name="define_int_context_menu.android"></string>
```

Comment:
I don't think using a "." is a good idea. I haven't tried it to be honest. But til now there were non mit "."'s  :D